eventstream.go

```diff
@@ -356,4 +362,267 @@ type Client interface {
 	Publish(publishBuilder *PublishBuilder) error
 	PublishSync(publishBuilder *PublishBuilder) error
 	Register(subscribeBuilder *SubscribeBuilder) error
+	PublishAuditLog(auditLogBuilder *AuditLogBuilder) error
+}
+
+type AuditLog struct {
+	ID              string           `json:"id,omitempty"`
+	Category        string           `json:"category,omitempty"`
+	ActionName      string           `json:"actionName,omitempty"`
+	Timestamp       int64            `json:"timestamp,omitempty"`
+	IP              string           `json:"ip,omitempty"`
+	Actor           string           `json:"actor,omitempty"`
+	ActorType       string           `json:"actorType,omitempty"`
+	ClientID        string           `json:"clientId,omitempty"`
+	ActorNamespace  string           `json:"actorNamespace,omitempty"`
+	ObjectIDs       []string         `json:"objectIds,omitempty"`
```

Comment:
here will we introduce the objectIds and targetUserIds, how 1:1 match the objectId with the targetUserId if there're multiple objects and users? 
or these 2 fields has no logic meaning, only for persistence and query?